services/clamav/src/index.test.js

```diff
@@ -0,0 +1,17 @@
+const uploadAV = require("./index");
+
+describe("clamav", () => {
+  it("should scan uploaded document with negative result", async () => {
+    const file = {
+      tempFilePath: "./eicar.com.txt",
+    };
+
+    const succesfullyUploaded = await uploadAV(file, {
+      fileName: "eicar.com.txt",
+      location: "./eicar.com.txt",
+      debug: true,
+    });
+
+    expect(succesfullyUploaded);
+  });
+});
```

Comment:
I guess it may seem obvious but shouldn't we also have a test for a positive result?

Reply:
Your right, this was missed out.